magenta-lib/src/test/scala/magenta/deployment_type/S3ObjectPrefixParametersTest.scala

```diff
@@ -32,6 +24,19 @@ class S3ObjectPrefixParametersTest extends AnyFlatSpec with Matchers {
     Stack("interactives"),
     Region("eu-west-1")
   )
+
+  val targetStageCODE = DeployTarget(
+    fixtures.parameters(stage = Stage("CODE")),
+    Stack("interactives"),
+    Region("eu-west-1")
+  )
+
+  val targetStageUnknown = DeployTarget(
```

Comment:
This `val` is only used in one test, should we co-locate it with that test?

Reply:
good shout, will do!